browser/brave_wallet/brave_wallet_provider_delegate_impl_helper_android.cc

```diff
@@ -4,17 +4,28 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/browser/brave_wallet/brave_wallet_provider_delegate_impl_helper.h"
+#include "brave/browser/brave_wallet/brave_wallet_service_factory.h"
+#include "brave/components/brave_wallet/browser/brave_wallet_service.h"
+#include "content/public/browser/web_contents.h"
 
 #include "base/notreached.h"
 
 namespace brave_wallet {
 
 void ShowPanel(content::WebContents* web_contents) {
-  NOTIMPLEMENTED();
+  BraveWalletService* brave_wallet_service =
+      brave_wallet::BraveWalletServiceFactory::GetServiceForContext(
+          web_contents->GetBrowserContext());
+  DCHECK(brave_wallet_service);
+  brave_wallet_service->ShowPanel();
 }
 
 void ShowWalletOnboarding(content::WebContents* web_contents) {
-  NOTIMPLEMENTED();
+  BraveWalletService* brave_wallet_service =
+      brave_wallet::BraveWalletServiceFactory::GetServiceForContext(
+          web_contents->GetBrowserContext());
+  DCHECK(brave_wallet_service);
```

Comment:
Better to either `CHECK` here or have an if around `brave_wallet_service->ShowWalletOnboarding();`

Reply:
shouldn't we be checking for null here instead of `CHECK` or `DCHECK`? 
```if (!IsAllowedForContext(context)) {
    return nullptr;
  }```

Reply:
sorry, not to imply we should be calling IsAllowedForContext there, I'm asking about a nullptr check because the service can return nullptr with that code

Reply:
I'm going to do that in that follow up https://github.com/brave/brave-browser/issues/22056